src/cpp/rtps/builtin/discovery/endpoint/EDPServer2.hpp

```diff
@@ -104,6 +111,12 @@ class EDPServer2 : public aux::EDPSimple
      */
     virtual bool createSEDPEndpoints() override;
 
+    //! EDP publications writer filter
+    IReaderDataFilter* edp_publications_filter_;
+
+    //! EDP subscriptions writer filter
+    IReaderDataFilter* edp_subscriptions_filter_;
+
```

Comment:
Why should we keep the interfaces references? Once passed to the Writers they are no longer useful.

Reply:
You're right!